sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -271,7 +271,7 @@
   /**
    * @since 2.11
    */
-  String ORGANISATION = "sonar.organisation";
+  String ORGANIZATION = "sonar.organisation";
```

Comment:
It's not possible to update an API like this... You need to deprecate ORGANISATION, and create a new ORGANIZATION parameter